File: Keepr/Models/VaultKeep.cs
namespace Keepr.Models
{
  public class VaultKeep
  {
    public int Id { get; set; }
    public int VaultId { get; set; }
    public int KeepId { get; set; }
    public string CreatorId { get; set; }
  }
  public class VaultKeepViewModel : Keep
  {
    public int KeepId { get; set; }
    public int VaultKeepId { get; set; }
  }
}

File: Keepr.client/src/models/VaultKeep.js
export class VaultKeep
{
  constructor (data= {}){
    this.id = data.id
    this.creatorId = data.creatorId
    this.profileId = data.profileId 
    this.accountId = data.accountId
    this.vaultKeepId = data.vaultKeepId
    this.keepId = data.keepId
    this.name = data.name
    this.img = data.img
    this.views = data.views
  }
}